### DISCUSSION TITLE
Data for the last 4 Months in GET_FLAT_FILE_ALL_ORDERS_DATA_BY_LAST_UPDATE_GENERAL

### DISCUSSION THREAD

**Discussion by ASchmidtGit** [amzn/selling-partner-api-models#3505] [General]
Heyho,

i need the orders for the last 25 Months. (item, price, amount, shippment, name, sku...)
If i pull GET_FLAT_FILE_ALL_ORDERS_DATA_BY_LAST_UPDATE_GENERAL i get data only up to 2022-12-26 but nothing for the current year.
Is this report limited? 
Even if i use "createdSince" or "dataStartTime" with 2023-01-01 i get nothing from this year

And another question: Is it possible to get data older than 90 days from "GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE"?

**Comment by JoeChan-develop** [comment]
Hi, I met the same problem, have you solve yet?

**Comment by MeenaAmz** [comment]
@JoeChan-develop Can you please elaborate on your issue? You should be able to get current data by setting dateStartTime to desirable date time for GET_FLAT_FILE_ALL_ORDERS_DATA_BY_LAST_UPDATE_GENERAL report. Note that this value is in [ISO 8601](https://developer-docs.amazon.com/sp-api/docs/iso-8601) date-time format. This [Tutorial](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-tutorial-request-a-report#step-1-request-a-report) can help with passing the correct parameters. Let me know if you have further questions.

Thanks
Meena
Selling Partner Developer Services

**Reply by JoeChan-develop** [reply]
@MeenaAmz 
Hi,mate.
Our requirement is that fetch seller's 2-year SP data, these detailed fields are: 1. 2-year settlement amount; 2. 2-year refund amount; 3. 2-year refund count(how many refund orders); And we been try to get it by GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2(Settlement Reports), but it returned error like "the request is more than 90 days old", so we are have no way now, don't know any other API can make it. 

What's more, can I have yours email address if you are available? Thanks .

**Reply by JoeChan-develop** [reply]
![response](https://github.com/user-attachments/assets/542854ea-6d0e-42ac-b354-a08794fdd3e8)

**Reply by JoeChan-develop** [reply]
@MeenaAmz 
Sorry mate I still got one more question, if we want to know a seller's category whether has digital product, how can we ensure this information by which API?
